.github/workflows/pylint.yml

```diff
@@ -0,0 +1,22 @@
+name: Pylint
+
+on:
+        push:
+                branches:
+                        - master
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Analysing the code with pylint
+      run:
+        python -m pylint --fail-under=9 `ls -R|grep .py$|xargs`
```

Comment:
I think this might not work as expected for hierarchy of folders. It works only if all python files are in the current/base directory. I tried it out, just to check if it works and the build failed with module not found:

https://github.com/qxf2/newsletter_automation/runs/2820064385?check_suite_focus=true

Reply:
I have modified the yml to use to a different command to get the .py files from child directories